code-challenges.test.js

```diff
@@ -13,34 +13,131 @@
+    expect(onlyNumbers(hodgepodge2)).toEqual([2, 1, -1])
+  })
+})
 
 // b) Create the function that makes the test pass.
+//Puesdo Code:
+//Create the function called onlyNumbers to match the test
+// Give the test one parameter called "arr" to match the data being passed in later as an argument.
+// Use the .filter method to iterate through the array and check the typeof value that is in each index.
+// make the typeof === 'number' so only numbers are in the new array.
+// Stack .map() to the end of that to iterate again through this new array.
+// take the value % 3 to find the remainder of the numbers.
+// return the new array.
+
+const onlyNumbers = (arr) => {
+  const newArray = arr
+    .filter((value) => typeof value === "number")
+    .map((num) => num % 3)
+  return newArray
+}
+
+// PASS  ./code-challenges.test.js
+//   onlyNumbers
+//     ✓ Function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.  (1 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
-
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
+describe("cubeItAddIt", () => {
+  const cubeAndSum1 = [2, 3, 4]
+  // Expected output: 99
+  const cubeAndSum2 = [0, 5, 10]
+  // Expected output: 1125
+  it("Function that takes in an array of numbers and returns the sum of all the numbers cubed", () => {
+    expect(cubeItAddIt(cubeAndSum1)).toEqual(99)
+    expect(cubeItAddIt(cubeAndSum2)).toEqual(1125)
+  })
+})
 
 // b) Create the function that makes the test pass.
+// Puesdo code
+// Create the function cubeItAddIt to match the test.
+// Give the function one parameter 'arr' to match the data that will be passed in.
+// Create a variable called 'sum' that is set to 0 for later
+// Create a new variable that will store the .map method to iterate through the array cubing each number
+// Attach the .forEach method to the new array variable to perform an action on each one of the values in the array.
+// inside of .forEach name the value 'number' to clean up the code.
+// use the sum varialbe made in the beginng with the shorthand += to add the number value.
+// return the sum.
+
+const cubeItAddIt = (arr) => {
+  let sum = 0
+  const newArray = arr.map((num) => num ** 3)
+  newArray.forEach(number => {
+    sum += number
+  })
+  return sum
+}
+
+// PASS  ./code-challenges.test.js
+// onlyNumbers
+//   ✓ Function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.  (1 ms)
+// cubeItAddIt
+//   ✓ Function that takes in an array of numbers and returns the sum of all the numbers cubed
+
+// Test Suites: 1 passed, 1 total
```

Comment:
Fantastic work here, Spencer!  Very clean, efficient code.  Your pseudocode shows you understand the concepts and how to implement the different methods very well.  Great job!  Congrats!